### PR TITLE
Updated id widget

### PR DESCRIPTION
**Pull request by varsha766**
## Title
Provide a short, descriptive title of the change (50 characters max recommended).
- Added toggel to disable widget login
---

## Summary
One-paragraph summary describing what the change does and why. Link related issue(s) or design docs.

---

## Background / Motivation
Explain the context, problem being solved, and any alternatives considered. Include links to design docs, issues, or RFCs.

---

## Scope of Changes
- Files / modules affected (high level)
- Public API changes (if any)
- Database / schema / migration impact (if any)

---

## Type
- [ ] Feature
- [ ] Bugfix
- [ ] Docs
- [ ] Refactor
- [ ] Tests
- [ ] Performance
- [ ] Build / CI / Config

---

## Implementation
Describe the approach taken, key design decisions, and why.

---

## Testing & QA
- Units / integration tests added: describe coverage and important test cases
- Manual / exploratory test steps for QA
- CI matrix: platforms/versions validated

---

## Migration / Rollout Plan
- Data migrations and downtime expectations
- Backwards compatibility notes
- Feature flags and staged rollout instructions

---

## Security / Privacy Considerations
List any security implications, new secrets, or compliance-related items.

---

## Performance Impact
Any benchmark results or expected performance regressions/improvements.

---

## Release Notes (for changelog)
One-line summary suitable for release notes / CHANGELOG.

---

## Checklist (required)
- [ ] Linked issue or ticket present (link: )
- [ ] Target branch is correct
- [ ] PR description explains the why, not just the what
- [ ] Code compiles and tests pass locally
- [ ] Unit and integration tests added where applicable
- [ ] Documentation updated (README, docs site, comments)
- [ ] CHANGELOG.md (or repo-specific changelog) updated with an entry
- [ ] `package.json` / package version updated if publishing a release
- [ ] No secrets or credentials committed
- [ ] Security review completed if required
- [ ] Performance benchmarks included if applicable
- [ ] Migration steps included if applicable

---

## Reviewer Guidance
List the specific areas you want reviewers to focus on (e.g., security, migration, public API, edge cases). Provide commands or steps to run the project locally to validate the change:

```bash
# checkout PR branch
git checkout $BRANCH
# run tests
npm ci && npm test
# run lint
npm run lint
```

---

## Acceptance Criteria
Define the minimal conditions that must be true for this PR to be merged (e.g., tests pass, performance threshold met, migration completed).

---

## Approvals
- [ ] At least one approving review from code owners
- [ ] CI status checks are green
- [ ] Any required manual QA sign-off

### CODE REVIEW

File: src/views/playground/WidgetConfig/Index.vue [hunk 1]
```diff
@@ -539,6 +557,10 @@ export default {
            label: "Enable Email Notifications",
            description: "Notify users via email regarding the status of their ID verification. When enabled, users will receive automated updates upon the successful completion or rejection of their verification attempt."
         },
+        widgetLogin: {
+          label: "Use ID Widget Login",
```
Comment (Vishwas1) [inline src/views/playground/WidgetConfig/Index.vue]: Enable Built-in Login & Identity Vault

File: src/views/playground/WidgetConfig/Index.vue [hunk 2]
```diff
@@ -539,6 +557,10 @@ export default {
...
+          description: "Let the ID widget handle user login. Turn this off when your application handles login itself."
```
Comment (Vishwas1) [inline src/views/playground/WidgetConfig/Index.vue]: Users sign in through the ID Widget and receive a personal identity vault for storing reusable credentials. Disable this if your application handles authentication.

**Review by Vishwas1 — CHANGES_REQUESTED** [review]

File: src/views/playground/WidgetConfig/Index.vue [hunk 3]
```diff
@@ -664,6 +688,9 @@ export default {
         .join(',')
     },
     validateField() {
```
Comment (varsha766) [inline src/views/playground/WidgetConfig/Index.vue:695]: @Vishwas1  This is the code where I am assigning the value of isWidgetLogin to isVaultEnabled. This is getting called from saveConfiguration and updateConfiguration widget

Comment (Vishwas1) [reply src/views/playground/WidgetConfig/Index.vue:695]: This is not right way to set fields. The responsiblity of validateField() is to "Validate" the field -- literelly what it says not to mutate field values.

I suggest to use watcher here to mutate isVaultEnabled field with whateever value

Comment (varsha766) [reply src/views/playground/WidgetConfig/Index.vue:695]: ok. I'll update

Comment (Vishwas1) [reply src/views/playground/WidgetConfig/Index.vue:695]: We already have watcher in the file

```
watch: {
    'widgetConfigTemp.zkProof.enabled':
    {
     ...
    },
  },
```

similarly add another watch to monitor the isisWidgetLogin field and mutate the

File: src/views/playground/WidgetConfig/Index.vue [hunk 4]
```diff
@@ -391,6 +405,12 @@ export default {
       },
       deep: true
     },
```
Comment (Vishwas1) [inline src/views/playground/WidgetConfig/Index.vue:407]: Add a comment on top 

//  Sync: Vault is only enabled when widget login is enabled

**Review by Vishwas1 — APPROVED** [review]
